hummingbot/connector/exchange/altmarkets/altmarkets_user_stream_tracker.py

```diff
@@ -35,13 +33,14 @@ def __init__(self,
                  throttler: Optional[AsyncThrottler] = None,
                  altmarkets_auth: Optional[AltmarketsAuth] = None,
                  trading_pairs: Optional[List[str]] = []):
-        super().__init__()
         self._altmarkets_auth: AltmarketsAuth = altmarkets_auth
         self._trading_pairs: List[str] = trading_pairs
-        self._ev_loop: asyncio.events.AbstractEventLoop = asyncio.get_event_loop()
-        self._data_source: Optional[UserStreamTrackerDataSource] = None
-        self._user_stream_tracking_task: Optional[asyncio.Task] = None
         self._throttler = throttler or AsyncThrottler(Constants.RATE_LIMITS)
+        super().__init__(data_source=AltmarketsAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by `super().__init__`. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.